fdbserver/workloads/WatchesSameKeyCorrectness.actor.cpp

```diff
@@ -57,13 +61,18 @@ struct WatchesSameKeyWorkload : TestWorkload {
 		return ok;
 	}
 
+	static int getId() {
```

Comment:
Can you please add some documentation why this is needed?

Reply:
removed this